cmake/FindCMSIS.cmake

```diff
@@ -96,9 +109,11 @@ ELSEIF(STM32_FAMILY STREQUAL "L1")
         SET(STM32Cube_DIR "/opt/STM32Cube_FW_L1_V1.8.0")
         MESSAGE(WARNING "No STM32Cube_DIR specified, using default: " ${STM32Cube_DIR})
     ENDIF()
+
```

Comment:
Those changes might be better of on another PR since they are not related to the support of the G0 serie